app/src/main/java/org/piramalswasthya/cho/ui/home/OfflineDataSharingBottomSheet.kt

```
@@ -0,0 +1,579 @@
+                        connectionsClient.stopAllEndpoints()
+                    }
+                }
+                onComplete(success)
+            }
+        }
+    }
+
+    private val payloadCallback = object : PayloadCallback() {
+        override fun onPayloadReceived(p0: String, p1: Payload) {
+
+        }
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)